cmake/Modules/DefinePythonSitePackages.cmake

```diff
@@ -23,23 +23,14 @@ function(find_site_packages pythonexecutable targetname)
 endfunction(find_site_packages)
 
 # Find default Python
-# When WITH_PYTHON3_DEFAULT is enabled, then require Python 3.
-if (WITH_PYTHON3_DEFAULT)
-    if (NOT WITH_PYTHON3)
-        message(FATAL_ERROR "WITH_PYTHON3_DEFAULT=ON requires WITH_PYTHON3=ON")
-    endif(NOT WITH_PYTHON3)
-    # find Python interpreter
-    set(Python_ADDITIONAL_VERSIONS 3)
-    find_package(PythonInterp REQUIRED)
-    # FindPythonInterp doesn't restrict version with ADDITIONAL_VERSIONS
-    if (PYTHON_VERSION_STRING VERSION_LESS "3.5.0")
-        message(FATAL_ERROR "Detect Python interpreter < 3.5.0")
-    endif()
+set(Python_ADDITIONAL_VERSIONS 3)
```

Comment:
What's the use of this variable? Also, the var name uses mixed cases. Should we follow same cases?

Reply:
I don't actually know, so I left it unchanged. This line came from line 32 in the original code. All I did was removing the checks for WITH_PYTHON3 and WITH_PYTHON3_DEFAULT since we will always use Python 3 anyway.

Reply:
Perhaps, we can remove it. Probably, through another patch.

Reply:
It's actually used to find the Python interpreter:
https://cmake.org/cmake/help/latest/module/FindPythonInterp.html
So we probably should keep it.